### ISSUE TITLE
[BUG] `MapController.move` does not emit `MapEventMove` with passed `id` argument

### ISSUE THREAD

**Issue by hidea** [fleaflet/flutter_map#1542]
### What is the bug?

MapController move with id is not reflected in the event. 

### How can we reproduce it?

```
mapController.move(LatLng(position.latitude, position.longitude),10, id: "current");
```
Event callback id contains null.
```
FlutterMap(
  ...
  options: MapOptions(
    onMapEvent: (event) {
      if (event is MapEventMove) {
        // event.id is null
      }
    }
  )
}
```

### Do you have a potential solution?

Perhaps the passing of the id is missing here?
https://github.com/fleaflet/flutter_map/blob/64e3a8ec304b999f9ef6d0bece7874216ff76bed/lib/src/map/flutter_map_state.dart#L385-L392

### Platforms

Web, iOS

### Severity

Minimum: Allows normal functioning

**Comment by ibrierley** [comment]
I think maybe this is fixed by https://github.com/fleaflet/flutter_map/pull/1534

**Comment by JaffaKetchup** [comment]
Thanks for reporting. This is fixed in #1532 (434ce32), which was originally from #1534. It should appear in the next v5 (pre?)release.
